corehq/motech/value_source.py

```diff
@@ -52,6 +67,22 @@ def deserialize(self, external_value):
     def get_value(self, case_trigger_info):
         raise NotImplementedError()
 
+    def check_direction(self, direction):
+        """
+        Checks whether the ValueSource direction allows the value to be
+        imported or exported.
+
+        >>> value_source = ValueSource(direction=DIRECTION_BOTH)
+        >>> bool(value_source.check_direction(DIRECTION_EXPORT))
+        True
+
+        >>> value_source = ValueSource(direction=DIRECTION_IMPORT)
+        >>> bool(value_source.check_direction(DIRECTION_EXPORT))
+        False
+
+        """
+        return DIRECTIONS[self.direction] & DIRECTIONS[direction]
```

Comment:
the logic with bytes is pretty cool. 
Though might be worth to just keep it simple 
`self.direction && direction == self.direction`
and get rid of 
```
DIRECTION_BOTH = None
DIRECTIONS = {
    DIRECTION_IMPORT: 0b01,
    DIRECTION_EXPORT: 0b10,
    DIRECTION_BOTH: 0b11,
}
```

Reply:
Yeah, that sounds fair. The bytes approach was based on what we did with the 99dots integration, but I agree this is simpler.